ApplicationLibCode/ReservoirDataModel/RigCaseCellResultsData.cpp

```diff
@@ -1987,6 +2000,112 @@ void RigCaseCellResultsData::computeDepthRelatedResults()
+
+    bool computeIndexI = false;
+    bool computeIndexJ = false;
+    bool computeIndexK = false;
+
+    if ( iResultIndex == cvf::UNDEFINED_SIZE_T )
+    {
+        iResultIndex  = this->addStaticScalarResult( RiaDefines::ResultCatType::STATIC_NATIVE,
+                                                    RiaResultNames::indexIResultName(),
+                                                    false,
+                                                    reservoirCellCount );
+        computeIndexI = true;
+    }
+
+    if ( jResultIndex == cvf::UNDEFINED_SIZE_T )
+    {
+        jResultIndex  = this->addStaticScalarResult( RiaDefines::ResultCatType::STATIC_NATIVE,
+                                                    RiaResultNames::indexJResultName(),
+                                                    false,
+                                                    reservoirCellCount );
+        computeIndexJ = true;
+    }
+
+    if ( kResultIndex == cvf::UNDEFINED_SIZE_T )
+    {
+        kResultIndex  = this->addStaticScalarResult( RiaDefines::ResultCatType::STATIC_NATIVE,
+                                                    RiaResultNames::indexKResultName(),
+                                                    false,
+                                                    reservoirCellCount );
+        computeIndexK = true;
+    }
+
+    std::vector<std::vector<double>>& indexI = m_cellScalarResults[iResultIndex];
+    std::vector<std::vector<double>>& indexJ = m_cellScalarResults[jResultIndex];
+    std::vector<std::vector<double>>& indexK = m_cellScalarResults[kResultIndex];
+
+    {
+        if ( indexI[0].size() < reservoirCellCount )
+        {
+            indexI[0].resize( reservoirCellCount, std::numeric_limits<double>::infinity() );
+            computeIndexI = true;
+        }
+
+        if ( indexJ[0].size() < reservoirCellCount )
+        {
+            indexJ[0].resize( reservoirCellCount, std::numeric_limits<double>::infinity() );
+            computeIndexJ = true;
+        }
+
+        if ( indexK[0].size() < reservoirCellCount )
+        {
+            indexK[0].resize( reservoirCellCount, std::numeric_limits<double>::infinity() );
+            computeIndexK = true;
+        }
+    }
+
+    for ( size_t cellIdx = 0; cellIdx < m_ownerMainGrid->globalCellArray().size(); cellIdx++ )
+    {
+        const RigCell& cell = m_ownerMainGrid->globalCellArray()[cellIdx];
+
+        size_t resultIndex = cellIdx;
+        if ( resultIndex == cvf::UNDEFINED_SIZE_T ) continue;
+
+        bool isTemporaryGrid = cell.hostGrid()->isTempGrid();
+
+        size_t       gridLocalNativeCellIndex = cell.gridLocalCellIndex();
+        RigGridBase* grid                     = cell.hostGrid();
+
+        size_t i, j, k;
+        if ( !isTemporaryGrid && grid->ijkFromCellIndex( gridLocalNativeCellIndex, &i, &j, &k ) )
+        {
+            if ( computeIndexI || isTemporaryGrid )
+            {
+                indexI[0][resultIndex] = i;
```

Comment:
@magnesj Should this be 1-indexed? It is currently 0-indexed, but user can see the result and is maybe the other option?

Reply:
Must be 1-based

Reply:
Fixed.